lib/screens/v2/widget_instance/dup_alert/serialize.ex

```diff
@@ -62,62 +70,74 @@ defmodule Screens.V2.WidgetInstance.DupAlert.Serialize do
     end
   end
 
-  defp get_line_text_builder(t) do
-    case DupAlert.get_affected_lines(t) do
-      [line_color] ->
-        fn _ -> [bold("#{line_color} Line")] end
+  defp line_to_pill_atom(line_string) do
```

Comment:
I think some of these functions would be good as utils, but instead of figuring that out here, I'd say that can go in the "alert refactor" task, since it'll be easier to find a place for this util while also figuring out where to put all the other common code, etc.